### PR TITLE
FLW-1072 iOS: Change custom validators. Search. Disable Done untill valid state

### PR DESCRIPTION
**Pull request by stas-litvinov**
fix Done button state

### CODE REVIEW

File: fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift [hunk 1]
```diff
@@ -69,22 +81,17 @@ final class ValidatorSearchViewController: UIViewController, ViewHolder, Importa
...
-        )
-
         let attributes: [NSAttributedString.Key: Any] = [
             .foregroundColor: R.color.colorWhite()!,
             .font: UIFont.p0Paragraph
         ]
 
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .normal)
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .highlighted)
+        doneButton.setTitleTextAttributes(attributes, for: .normal)
```
Comment (ERussel) [inline fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift:89]: I believe that for validator filters and your validators we use `setupDefaultTitleStyle` of UIBarButtonItem to setup style. Should we apply it also here?

Comment (stas-litvinov) [reply fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift]: Fixed

File: fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift [hunk 2]
```diff
@@ -69,22 +81,10 @@ final class ValidatorSearchViewController: UIViewController, ViewHolder, Importa
...
-            .font: UIFont.p0Paragraph
-        ]
-
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .normal)
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .highlighted)
-
-        navigationItem.rightBarButtonItem = rightBarButtonItem
+        doneButton.setupDefaultTitleStyle(with: .p0Paragraph)
+        doneButton.isEnabled = false
+z
```
Comment (ERussel) [inline fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift:86]: Looks like Cmd+Z didn't work :)

**Review by ERussel — APPROVED** [review]

File: fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift (hunk 2 above)
Comment (stas-litvinov) [reply fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift:86]: Ouch.. Actually, my left Cmd button got broken ):

**Pull request merged**